### PR TITLE
About page hero

### PR DESCRIPTION
**Pull request by kevinjihwanlee**
This is the implementation of the hero banner in the About Page. 

Also included in this PR:
-  Adjustment of the TopBar such that it sticks to the top of page regardless of where you scroll, which required a slight adjustment of styling in other parts of the application.
- Slight change in the Webpack config to allow imports of larger static image files.

### CODE REVIEW

**Comment by kevinjihwanlee** [comment]
image

**Comment by kevinjihwanlee** [comment]
[About page hero](https://trello.com/c/j5lRHAI2/149-about-page-hero)

File: src/components/App.test.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { UnconnectedApp } from './App';
 
 jest.mock('util/firebase');
 jest.mock('images/facebook-logo.png');
+jest.mock('images/Cover1.jpg');
```
Comment (Joonpark13) [inline src/components/App.test.js]: Let's rename the image to something more descriptive.

File: src/components/pages/AboutPage.js [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+    backgroundAttachment: 'fixed',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+  },
+  heroText: {
+    position: 'absolute',
+    top: '0px',
+    left: '0px',
+    height: '600px',
```
Comment (Joonpark13) [inline src/components/pages/AboutPage.js]: This 600 is the same value as the `minHeight` for the `hero` class, right? Let's extract this into a variable and use it in both places - take advantage of the power of JSS!

Comment (kevinjihwanlee) [reply src/components/pages/AboutPage.js]: great point, resolved

File: src/components/App.test.js (hunk 1 above)
Comment (kevinjihwanlee) [reply src/components/App.test.js]: renamed to `hero-image.jpg`, resolved

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**